src/components/panel/panel.js

```diff
@@ -1663,6 +1798,23 @@ MdPanelRef.prototype._done = function(callback, self) {
 };
 
 
+/**
+ * Adds a panel to a group if the panel does not already exist within the group.
+ * The only way for a panel to be apart of a group and then to be removed from
```

Comment:
Well this seems suspicious... Should we add a removeFromGroup method? I think it follows the same use case/user journey as dynamic adding.